.circleci/config.yml

```diff
@@ -994,25 +994,49 @@ workflows:
                 at: /
           workflow-name: "nightly-zero-stake-regression"
```

Comment:
Please add link of the passed test to the PR. So, it will be easy to refer in the future.

Reply:
Not sure what this exactly means. Is the URL for the test running in CircleCi: 
https://app.circleci.com/pipelines/github/hashgraph/hedera-services/2972/workflows/81ac55af-8413-4876-a6b1-312f4198ad34 for mainnet migration test and
https://app.circleci.com/pipelines/github/hashgraph/hedera-services/2972/workflows/b23155cf-b52a-4cf2-8271-0ddc8d5c6b9b for public testnet migration test?

Reply:
Sorry for not being clear. I mean you can add link of the passed test posted to slack to the PR , just to show this configuration worked.